src/components/Table/body/actionsCell.js

```diff
@@ -25,7 +25,7 @@ function MenuItems({ children, rowData }) {
 }
 
 export default function ActionsCell(props) {
-    const { columnChildren, rowsLength, rowIndex, rowData } = props;
+    const { columnChildren, rowsLength, rowIndex, rowData, variant } = props;
```

Comment:
instead of passing down variant prop to all these components, we can wrap the Table index.js with the ThemeProvider from styled-components and pass the variant in the context, thus al styled below the table will receive the variant prop
Look here: https://codesandbox.io/s/delicate-lake-ew9hu?file=/src/App.js:0-671